config/dev.yml

```diff
@@ -1,3 +1,6 @@
 ---
 dev_name: signalfx-bridge
```

Comment:
No, the internal name for it is still *signalfx-bridge*.  I just renamed the github repo for easier management of the org.